src/main/java/org/apache/sling/caconfig/ConfigurationBuilder.java

```diff
@@ -86,4 +86,11 @@
      */
     @NotNull <T> Collection<T> asAdaptableCollection(@NotNull Class<T> clazz);
 
+    /**
+     * Check if the configuration node is existing
```

Comment:
Can you extend the javadoc to cover all aspects mentioned in https://issues.apache.org/jira/browse/SLING-8272?focusedCommentId=16766066&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-16766066

Reply:
Extended this with all the mentioned aspects.